akka-persistence/src/test/scala/akka/persistence/fsm/PersistentFSMActorSpec.scala

```diff
@@ -349,6 +399,20 @@ object PersistentFSMActorSpec {
     def props(persistenceId: String, reportActor: ActorRef) =
       Props(new WebStoreCustomerFSMActor(persistenceId, reportActor))
   }
+
+  class PersistentEventsStreamer(id: String, client: ActorRef) extends PersistentView {
```

Comment:
Let's make it a `PersistentActor`, as we'll try to remove `PersistentView` soon.